packages/connect-explorer/src/components/Header.tsx

```diff
@@ -60,23 +60,15 @@ const Header = () => (
         <HeaderPrimary>
             <HeaderItemsPrimary>
                 <a href="#/">
-                    <Svg
```

Comment:
can we use .svg file instead of this?

Reply:
if you show us how to.. 🤷

Reply:
add to suite-data, copy and import? but let's skip it :D